core/src/main/scala/com/codacy/analysis/core/git/Repository.scala

```diff
@@ -19,9 +20,11 @@ class Repository(repository: JGitRepository) {
     }
   }
 
-  def hasUncommittedChanges: Try[Boolean] = {
+  def uncommitedFiles: Try[Set[String]] = {
     Try {
-      jGit.status().call().hasUncommittedChanges
+      jGit.status().call().getUncommittedChanges.asScala.toSet ++
+        jGit.status().call().getUntracked.asScala.toSet ++
+        jGit.status().call().getUntrackedFolders.asScala.toSet
```

Comment:
Do all of the ".asScala", sum those and convert the sum to a ".toSet" if possible.